Gemfile

```diff
@@ -49,5 +49,5 @@ group :development, :test do
   gem 'byebug', '~> 11'
   gem 'listen', '~> 3'
   gem 'pry'
-  gem 'rack-mini-profiler', github: "MiniProfiler/rack-mini-profiler", require: false
+  gem 'rack-mini-profiler', require: false
```

Comment:
Sounds like you can also remove `git` from https://github.com/alphagov/govwifi-admin/blob/65580fd94da019c22b2fb0b51960bd90f5b27629/Dockerfile#L39, based on https://github.com/alphagov/govwifi-admin/commit/3bb369952064434c6427fb9269e489ed545129bb

Reply:
done :heavy_check_mark: